opentrons-ai-client/src/molecules/ModuleListItemGroup/index.tsx

```diff
@@ -133,10 +134,16 @@ export function ModuleListItemGroup(): JSX.Element | null {
                               )
                             },
                             dropdownType: 'neutral' as DropdownBorder,
-                            filterOptions: adapters?.map(adapter => ({
-                              name: getDefDisplayName(adapter),
-                              value: adapter,
-                            })),
+                            filterOptions: adapters
+                              ?.filter(adapter =>
+                                module.type === TEMPERATURE_MODULE_TYPE
+                                  ? adapter.includes('adapter')
+                                  : true
+                              )
+                              ?.map(adapter => ({
+                                name: getDefDisplayName(adapter),
+                                value: adapter,
+                              })),
```

Comment:
nit, this is a bit cleaner in my opinion

```suggestion
filterOptions: adapters
  ?.filter(adapter => {
    if (module.type === TEMPERATURE_MODULE_TYPE) {
      return adapter.includes('adapter')
    }
    return true
  })
  ?.map(adapter => ({
    name: getDefDisplayName(adapter),
    value: adapter,
  })),

```

Reply:
Changed, thanks!